resolwe/flow/tests/test_commands.py

```diff
@@ -141,3 +141,21 @@ def test_list_has_test_image_yaml(self):
         self.assertTrue(len(imgs) != 0)
         self.assertTrue(isinstance(imgs[0], dict))
         self.assertTrue(dict(name='resolwe/test', tag='base') in imgs)
+
+    def test_list_is_sorted(self):
+        # The returned list must be sorted alphabetically
+        out, err = StringIO(), StringIO()
+        call_command('list_docker_images', stdout=out, stderr=err)
+        self.assertEqual('', err.getvalue())
+        self.assertNotEqual('', out.getvalue())
+        lines = out.getvalue().split('\n')[:-1]
+        self.assertEqual(lines, sorted(lines))
```

Comment:
How do you make sure that output is not sorted by accident?

Reply:
I don't.  It's highly unlikely that this would happen by accident, even in the test environment with its small number of processes with Docker images, let alone in production.  Sorting of the output is a cosmetic feature only, anyways.